questions/tests/test_intervals.py

```diff
@@ -31,3 +32,24 @@ def test_for_correct_question(self):
         expected_question = "The interval C2 to G2 becomes which interval when C2 is lowered 1 " \
                             "octave(s) and G2 is raised 1 octave(s)?"
         self.assertEqual(expected_question, self.question.question)
+
+
+class TestInvertedInterval(TestCase):
+
+     def setUp(self):
+         self.question = InvertedInterval(interval_quality='Major')
+
+     def test_question(self):
+         expected_question = "When inverted, a Major interval becomes:"
+         self.assertEqual(expected_question, self.question.question)
+
+     def test_question_added_to_cache(self):
```

Comment:
We probably don't need this test since the caching happens in the super class and is tested elsewhere.